extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -178,16 +187,22 @@ export class AmpStoryReaction extends AMP.BaseElement {
   }
 
   /** @override */
-  buildCallback() {
+  buildCallback(concreteCSS = '') {
     this.options_ = this.parseOptions_();
     this.rootEl_ = this.buildComponent();
-    this.element.classList.add('i-amphtml-story-reaction');
+    this.rootEl_.classList.add('i-amphtml-story-reaction');
+    toArray(this.element.attributes).forEach((attr) => {
+      if (Object.values(ReactionFlags).includes(attr.name)) {
```

Comment:
This loops so many times over the ReactionFlags properties and the created array through `.includes`, can we just do one `Object.values` + one loop with a getAttribute?

Reply:
Sure good idea, I was relying on the flags being only 3-4 but to futureproof it, makes sense